client/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
```

Comment:
are you going to use this file ?

Reply:
This file has been removed in another pull request, due to commit issues it was shown in my commit.